rt_playground/dashboard.py

```diff
@@ -10,4 +12,7 @@ def application(environ, start_response):
 
 
 if __name__ == '__main__':
-    run_simple('localhost', 8080, application, use_reloader=True)
+    host = os.environ.get("HOST")
+    port = int(os.environ.get("PORT"))
+    use_reloader = bool(os.environ.get("USE_RELOADER", True))

```

Comment:
found a bug in my own code ^^
bool("0") evaluates to True :)
```suggestion
    use_reloader = bool(int(os.environ.get("USE_RELOADER", "1")))
```